### PR TITLE
Beautify

### PR DESCRIPTION
**Pull request by Olamarx**
- Beautify the page

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -19,25 +19,29 @@ body {
 .header__h1 {
   font-size: 4rem;
   margin: 30px 40px;
+  transition: all 0.4s ease;
 }
 
 .main-body {
```
Comment (rloterh) [inline src/style.css]: The layout of your app does not match the provided [wireframe](https://github.com/microverseinc/curriculum-javascript/blob/main/leaderboard/images/leaderboard_wireframe.png). Kindly note we as much as possible want our app to follow the wireframe template as much as possible. Kindly adjust the following aspects of your design:
- [x] _Recent scores_ text, _Refresh_ button, and _Add your scrore_ text should be aligned horizontally, as seen in the [wireframe](https://github.com/microverseinc/curriculum-javascript/blob/main/leaderboard/images/leaderboard_wireframe.png) template
- [x] the _Refresh_ button should be aligned to the right as the scores table
- [x] the _Submit_ button should be aligned to the right as the form input fields

**Your design:**
image


**The wireframe design:**
image

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Olamarx, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 🚦 🚦 🚦  

## Highlights 👍  
- App is running as intended

## Required Changes ♻️
_Kindly check the comments under the review._
- [x] Kindly ensure to add instructions on how to run your application, so as to make your ReadMe appear more professional

<br>
Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/style.css (hunk 1 above)
Comment (Olamarx) [reply src/style.css]: @rloterh the readme file has how-to instructions on how to run it on the local machine and that is `Cloning the repository`.

**Review by Munsa1 — APPROVED** [review]
Hi @Olamarx, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
:heavy_check_mark: Descriptive Readme
:heavy_check_mark: Passing Linters
:heavy_check_mark: Correct Gitflow

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**